flink-streaming-scala/src/test/scala/org/apache/flink/streaming/api/scala/AsyncDataStreamITCase.scala

```diff
@@ -33,37 +32,39 @@ import scala.collection.mutable
 import scala.concurrent.{ExecutionContext, Future}
 
 object AsyncDataStreamITCase {
-  val timeout = 1000L
+  val timeout = 10000L
```

Comment:
These two variables can become local variables in the private methods (`testAsyncWaitTimeout` and `testAsyncWaitUsingAnonymousFunction`) that execute the tests. This will help the readability of the tests as the reader will not have to scroll up and down to see the values.